Source/WebKit/WebProcess/WebPage/mac/WebPageMac.mm

```diff
@@ -930,7 +930,14 @@ static void drawPDFPage(PDFDocument *pdfDocument, CFIndex pageIndex, CGContextRe
         if (RefPtr pluginView = static_cast<PluginView*>(embedOrObject->pluginWidget())) {
             if (pluginView->performImmediateActionHitTestAtLocation(locationInViewCoordinates, immediateActionResult)) {
                 // FIXME (144030): Focus does not seem to get set to the PDF when invoking the menu.
-                if (RefPtr pluginDocument = dynamicDowncast<PluginDocument>(element->document()))
+                RefPtr pluginDocument = dynamicDowncast<PluginDocument>(element->document());
+                auto shouldFocusPluginDocument = [&pluginDocument, &immediateActionResult] {
+                    if (!pluginDocument)
+                        return false;
+                    return !immediateActionResult.isActivePDFAnnotation;
+                }();
```

Comment:
Optional, if you want to be a little concise. I don't think it harms readability too much:

```suggestion
                bool shouldFocusPluginDocument = pluginDocument ? !immediateActionResult.isActivePDFAnnotation : false;
```